src/Symfony/Component/Process/Tests/PipeStdinInStdoutStdErrStreamSelect.php

```diff
@@ -32,7 +32,7 @@
         }
         $out = (binary) substr($out, $written);
```

Comment:
A build with changes already running 12 mins instead of 4, so I assume it some magic behind those binary casts.